src/main/java/de/doubleslash/keeptime/model/Settings.java

```diff
@@ -59,6 +59,38 @@
 
    private boolean hideProjectsOnMouseExit;
 
+   private double windowPositionX;

```

Comment:
may should start putting fields belonging together into own class like here (x,y,hash)